src/components/Profile/UploadImage.tsx

```diff
@@ -0,0 +1,56 @@
+import { uploadPhoto } from '@/utils/imageUpload';
+import { CircularProgress } from '@mui/material';
+import Image from 'next/image';
+import React, { useState } from 'react';
+
+type UploadImageProps = {
+    isEditMode: boolean;
+    setFreelancer: Function;
+    freelancer: any;
+}
+
+const UploadImage = ({ isEditMode, setFreelancer, freelancer }: UploadImageProps) => {
+    const [image, setImage] = useState<any>(freelancer?.profile_image)
+    const [loading, setLoading] = useState<boolean>(false)
+
+    const handleUpload = async (files: FileList | null) => {
+        if (files?.length) {
+            setLoading(true)
+            const data = await uploadPhoto(files[0])
+            if (data.url) {
+                setImage(data.url)
+                setFreelancer((prev: any) => {
+                    return { ...prev, profile_image: data.url }
+                })
+            }
+        }
+    }
+
+    return (
+        <div className="h-[160px] w-[160px] bg-[#2c2c2c] rounded-[100%] p-12 relative mt-[-120px] unset mx-auto">
```

Comment:
h-[160px]
w-[160px]
bg-[#2c2c2c]

Reply:
Most of the tailwind was pre implemented. I'll clean up asap